nodepool/files/nodepool.yml

```diff
@@ -192,6 +204,27 @@ providers:
         max-servers: 10
         availability-zones: []
         labels: *provider_pools_onmetal_labels_block
+      - name: snapshots
+        # Ignore the quota given by public cloud and only respect what we provide as max-servers
+        ignore-provider-quota: true
+        max-servers: 25
+        availability-zones: []
+        labels:
+          - name: rpc-r14-xenial_loose_artifacts-swift
+            cloud-image: rpc-r14.17.0-xenial_loose_artifacts-swift
```

Comment:
Oh yes. This provider will need to have its own cloud images block - it can't use the anchor. The block for this region will have to have a copy of the IAD one, but with the additional images that these labels will use.